k-distribution/tests/regression-new/array-haskell/1.test

```diff
@@ -1 +1 @@
-arr ( .List, 0, 0 ) [ 0 ]
+makeArray(0, 0) [ 0 ]
```

Comment:
What is going on here? Why doesn't it simplify down to the constructor anymore?

Reply:
This is the input, not the output. What is going on is we have made the internal implementation of arrays private, so we now have to go through the (public) array constructor.